doc-examples/example-groovy/build.gradle.kts

```diff
@@ -6,3 +6,11 @@ plugins {
 micronaut {
     testRuntime("spock")
 }
+
```

Comment:
@sdelamo I looked at this issue before, I think it started with gradle plugin 4.2.1, not sure why this was not needed before but now it looks like we need it and it fixes the build.